cfme/tests/cloud/test_providers.py

```diff
@@ -82,7 +82,7 @@ def test_provider_crud(provider):
         test_flag: crud
     """
     provider.create()
-    provider.validate(db=False)
+    provider.validate(ui=True)
```

Comment:
So, the issue with it is, that you can't make it take method, because not all the stats have all the methods. Therefore if you requested 'rest' num_datastores for infra would fail.